config/local/cloud-gateway-service.yml

```diff
@@ -4,6 +4,7 @@ apiml:
     service:
         forwardClientCertEnabled: true
         hostname: localhost
+        port: 10023
```

Comment:
Why? In application.yaml already exact same port defined:
https://github.com/zowe/api-layer/blob/da29403394731daeb62cabdbfdeeb000f9c47366/cloud-gateway-service/src/main/resources/application.yml#L36

Reply:
it's the local file, it's to be able to update the port without having to rebuild the jar. The other services have it also in their local yml files

Reply:
It is clear that this file is for local deployment, my point was that you are adding the property with the same value as in "inside jar" application.yml. I don't see the reason in it, because if someone what to change the port during local deployment, they will add the parameter themself.